### PR TITLE
adding documentation to the database files

### PR DESCRIPTION
**Pull request by torbjoernk**

### CODE REVIEW

File: _data/people.yml [hunk 1]
```diff
@@ -1,30 +1,63 @@
...
-# research interests, publications and contact data.
+# ## Identifier Format
 #
-# !! NOTE !!
-# Please keep the entries below in alphanumerical order according to the identifier. Thanks!
+# For the sake of uniqueness and uniformity, construct a person's identifier according to this
+# scheme:
+#
+#   SURNAME1_SURNAME2_GIVEN_NAME_INITIALS
+#
```
Comment (pancetta) [inline _data/people.yml:17]: Wait, this is not what we did so far!

File: _data/people.yml [hunk 2]
```diff
@@ -1,30 +1,63 @@
...
-# research interests, publications and contact data.
+# ## Identifier Format
 #
-# !! NOTE !!
-# Please keep the entries below in alphanumerical order according to the identifier. Thanks!
+# For the sake of uniqueness and uniformity, construct a person's identifier according to this
+# scheme:
+#
+#   SURNAME1_SURNAME2_GIVEN_NAME_INITIAL
+#
```
Comment (pancetta) [inline _data/people.yml:17]: Nope.. it's `SURNAME1_SURNAME2_INITIAL`, right?

Comment (torbjoernk) [inline _data/people.yml:17]: Just to be clear: `GIVEN_NAME_INITIAL` means "first character of the first given name".

Comment (pancetta) [inline _data/people.yml:17]: Aaaahh, I see.

**Pull request merged**